internal/mode/static/metrics/collectors/nginx.go

```diff
@@ -1,42 +1,35 @@
 package collectors
 
 import (
-	"context"
-	"fmt"
-	"net"
-	"net/http"
-
 	"github.com/go-kit/log"
 	"github.com/nginxinc/nginx-plus-go-client/client"
 	prometheusClient "github.com/nginxinc/nginx-prometheus-exporter/client"
 	nginxCollector "github.com/nginxinc/nginx-prometheus-exporter/collector"
 	"github.com/prometheus/client_golang/prometheus"
 
 	"github.com/nginxinc/nginx-gateway-fabric/internal/mode/static/metrics"
+	"github.com/nginxinc/nginx-gateway-fabric/internal/mode/static/nginx/runtime"
 )
 
 const (
-	nginxStatusSock  = "/var/run/nginx/nginx-status.sock"
-	nginxStatusURI   = "http://config-status/stub_status"
-	nginxPlusAPISock = "/var/run/nginx/nginx-plus-api.sock"
-	nginxPlusAPIURI  = "http://nginx-plus-api/api"
+	nginxStatusSock = "/var/run/nginx/nginx-status.sock"
+	nginxStatusURI  = "http://config-status/stub_status"
 )
 
 // NewNginxMetricsCollector creates an NginxCollector which fetches stats from NGINX over a unix socket
 func NewNginxMetricsCollector(constLabels map[string]string, logger log.Logger) prometheus.Collector {
```

Comment:
The reason I now pass the plus client to the plus collector is because the client is needed elsewhere. The regular nginx client is currently only used here, so not sure if there's much room for refactor in that sense.

Reply:
I could move the constants inside the function to prevent confusion.

Reply:
> I could move the constants inside the function to prevent confusion.

I don't think it will help much.

if it doesn't look like a consistency problem to you, the current code is all right then